### ISSUE TITLE
Does this bot handle reuploaded CDN images?

### ISSUE THREAD

**Issue by hbg** [xXAligatorXx/RepostCheckerBot#11]
If not, you can check out [https://github.com/harrisbegca/GeneralReposti](https://github.com/harrisbegca/GeneralReposti) and perhaps we can merge the two

**Comment by A1igator** [comment]
Sorry for the late reply, I've been very busy with school and have just started to get back into fixing the bot up, but it does not if I'm understanding what a CDN is correctly.

**Comment by hbg** [comment]
@A1igator oh... yeah basically I was wondering if it handled the same image uploaded from different image hosting sites (e.g. imgur, imgbb)

**Comment by A1igator** [comment]
That depends on whether praw can give the image to me or not. I have not explicitly added much logic for different image hosting sites. How do you do it? Use each image hosting site's api?

**Comment by hbg** [comment]
No, I used an image clustering library from Clarifai to determine the similarity between images - it'll upload every Reddit post's image to a central database that is then checked against for reposts.
